Tests/MapboxMapsTests/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandlerTests.swift

```diff
@@ -49,6 +51,13 @@ final class DoubleTapToZoomInGestureHandlerTests: XCTestCase {
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.camera, CameraOptions(zoom: mapboxMap.cameraState.zoom + 1))
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.duration, 0.3)
         XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.curve, .easeOut)
-        XCTAssertNil(cameraAnimationsManager.easeToStub.parameters.first?.completion)
+        XCTAssertNotNil(cameraAnimationsManager.easeToStub.parameters.first?.completion)
+        XCTAssertEqual(delegate.gestureEndedStub.parameters.first?.gestureType, .doubleTapToZoomIn)
+        let willAnimate = try XCTUnwrap(delegate.gestureEndedStub.parameters.first?.willAnimate)
+        XCTAssertTrue(willAnimate)
+
+        let didAnimateCompletion = try XCTUnwrap(cameraAnimationsManager.easeToStub.parameters.first?.completion)
```

Comment:
I think a better name for `didAnimateCompletion` would be `easeToCompletion`

Reply:
Same goes for the other similar tests. Let's name according to the type of animation that's completing